rocketpy/utilities.py

```diff
@@ -33,3 +39,96 @@ def compute_CdS_from_drop_test(
 
     CdS = 2 * rocket_mass * gravity / ((terminal_velocity**2) * air_density)
     return CdS
+
+
+def calculateEquilibriumAltitude(
+    rocket_mass, CdS, z0, v0=0, env=None, eps=1e-3, seeGraphs=True
+):
+    """Returns a dictionary containing the time, height and velocoty of the
+    system rocket-parachute in which the terminal velocoty is reached.
+
+    Parameters
+    ----------
+    rocket_mass : float
+        Rocket's mass in kg.
+    CdS : float
+        Number equal to drag coefficient times reference area for parachute.
+    z0 : float
+        Initial height of the rocket in meters.
+    v0 : float, optional
+        Rocket's initial speed in m/s.
+    env : Environment, optional
+        Environmental conditions at the time of the launch.
+    eps : float, optional
+        acceptable error in meters.
+    seeGraphs : boolean, optional
+        True if you want to see time vs height and time vs speed graphs,
+        False otherwise.
+
+
+    Returns
+    -------
+    infos : dictionary
+        Dictionary containing the values for time, height and speed of
+        the rocket when it reaches terminal velocity.
+    """
+
+    def check_constant(f, eps):
+        for i in range(len(f) - 2):
+            if abs(f[i + 2] - f[i + 1]) < eps and abs(f[i + 1] - f[i]) < eps:
+                return i
+
+        return None
+
+    g = -9.80665
+    if env == None:
+        environment = Environment(

```

Comment:
If you are not using Environment Atmospheric models more sophisticated than the default, than I suggest removing the optional arguments (i.e. lat, lon, date, etc.).

However, I believe it would be great if we use an environment object as input on this function, This way you first set an environment of your preference, and then you evaluate the optimal height for opening your parachute. I see you made that by seting env as optional, in my opinio you had the best approach